backend/src/modules/refresh-tokens/refresh-token.entity.ts

```diff
@@ -0,0 +1,52 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  Entity,
+  Column,
+  ManyToOne,
+  JoinColumn,
+  UpdateDateColumn,
+  CreateDateColumn,
+  BeforeInsert,
+  PrimaryColumn,
+} from 'typeorm';
+import { CommonEntity } from 'src/common/entities';
+import { UserEntity } from '../users/user.entity';
+import * as bcrypt from 'bcrypt';
+import { SALT_ROUNDS } from 'src/common/constants';
+
+@Entity({ name: 'refresh-tokens' })
```

Comment:
Remove id, createdAt and updatedAt fields from all entities, as these fields are declared in CommonEntity